dnf-behave-tests/features/steps/repo.py

```diff
@@ -298,3 +298,14 @@ def step_countme(context, quantifier):
         | path                     |
         | */metalink.xml?countme=1 |
     """.format(quantifier))
+
+
+@behave.step("I make packages from \"{repo}\" repository accessible via {rtype:repo_type}")
```

Comment:
Can you please phrase it like:
> I make packages from repository \"{repo}\" accessible ...

i.e. flip the "repo" and repository words, that way sounds better in english and is consistent with the other steps.